judge/models/interface.py

```diff
@@ -61,6 +64,11 @@ def pattern(self, cache={}):
             pattern = cache[self.regex] = re.compile(self.regex, re.VERBOSE)
             return pattern
 
+    @staticmethod
```

Comment:
Can we use `classmethod`? This is unpythonic, especially when you refer to the class in the body.